src/websocket/types.ts

```diff
@@ -254,12 +254,12 @@ export interface WsSubscribeTokenPrices extends WsSubscribeParams {
   denom: string;
 }
 
-export interface WsSubscribeAllTokenPrices extends WsSubscribeParams {}
+export type WsSubscribeAllTokenPrices = WsSubscribeParams
 export interface WsSubscribeCDPTokenSupplyByDenom extends WsSubscribeParams {
   denom: string;
 }
 
-export interface WsSubscribeCDPTokenSupply extends WsSubscribeParams {}
+export type WsSubscribeCDPTokenSupply = WsSubscribeParams
```

Comment:
Same as insights QueryXX structs, either eslint-disable this line or find the prop that does not allow empty structs.